content/id/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Membuat Load Balancer Eksternal
+content_template: templates/task
+weight: 80
+---
+
+
+{{% capture overview %}}
+
+Halaman ini menjelaskan bagaimana membuat _Load Balancer_ Eksternal.
+
+{{< note >}}
+Fitur ini hanya tersedia untuk penyedia _cloud_ atau lingkungan yang mendukung _load balancer_ eksternal.
+{{< /note >}}
+
+Ketika membuat servis, kamu mempunyai opsi untuk membuat jaringan _cloud load balancer_ secara otomatis.
+Hal ini menyediakan akses eksternal alamat IP yang dapat mengirim lalu lintas melalui porta yang tepat pada klaster _node_ kamu
+_asalkan klaster kamu beroperasi pada lingkungan yang mendukung dan terkonfigurasi dengan paket penyedia cloud load balancer yang benar_.
+
+Untuk informasi mengenai penyediaan dan penggunaan sumber daya _Ingress_ yang dapat memberikan
+servis URL yang dapat dijangkau secara eksternal, penyeimbang beban lalu lintas, terminasi SSL, dll.,
+silahkan cek dokumentasi [_Ingress_](/docs/concepts/services-networking/ingress/)
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Berkas konfigurasi
+
+Untuk membuat _load balancer_ eksternal, tambahkan baris dibawah ini ke
+[berkas konfigurasi servis](/docs/concepts/services-networking/service/#loadbalancer) kamu:
+
+```yaml
+    type: LoadBalancer
+```
+
+Berkas konfigurasi kamu mungkin terlihat seperti ini:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-service
+spec:
+  selector:
+    app: example
+  ports:
+    - port: 8765
+      targetPort: 9376
+  type: LoadBalancer
+```
+
+## Menggunakan kubectl
+
+Kamu dapat membuat servis dengan perintah `kubectl expose` dan
+_flag_ `--type=LoadBalancer`:
+
+```bash
+kubectl expose rc example --port=8765 --target-port=9376 \
+        --name=example-service --type=LoadBalancer
+```
+
+Perintah ini membuat servis baru dengan menggunakan pemilih yang sama dengan
```

Comment:
```suggestion
Perintah ini membuat Service baru dengan menggunakan pemilih yang sama dengan
```

Reply:
@kubernetes/sig-docs-id-reviews great way of translating "selector" -> "pemilih", instead of inventing new word "selektor"

wdyt?